.codecov.yml

```diff
@@ -1,12 +1,9 @@
-codecov:
-  comment: false
-  coverage:
-    status:
-      change:
-        enabled: false
-      patch:
-        enabled: false
-      project:
-        enabled: true
-        target: auto
-        threshold: 100%
+comment: false
```

Comment:
I hope, @stevepeak made this in a PR and he created codecov, let's give it a try?